src/renderer/npm.ts

```diff
@@ -162,3 +160,16 @@ export function packageRun(
 ): Promise<string> {
   return exec(dir, `${packageManager} run ${command}`);
 }
+
+export function decommentWithWorker(input: string): Promise<string> {
+  return new Promise((resolve, reject) => {
+    const worker = new Worker('../utils/decomment.ts');
```

Comment:
My only concern is that our build process will somehow destroy this. Did you run `npm run build` and check that it still works post compilation?

Reply:
`yarn package` seems to work fine for me. :)